scipy/linalg/tests/test_basic.py

```diff
@@ -1566,7 +1566,7 @@ def test_scaling(self):
         # Pre/post LAPACK 3.5.0 gives the same result up to an offset
         # since in each case col norm is x1000 greater and
         # 1000 / 32 ~= 1 * 32 hence balanced with 2 ** 5.
-        assert_allclose(int(np.diff(np.log2(np.diag(y)))), 5)
+        assert_allclose(int(np.diff(np.log2(np.diag(y)))[0]), 5)
```

Comment:
```suggestion
        assert_allclose(np.diff(np.log2(np.diag(y))), [5])
```
Would this work? If so it's a more strict test.

Reply:
Thanks for the suggestion! You're right, and even `assert_allclose(np.diff(np.log2(np.diag(y))), 5)` works, as long as there is no cast to int. I'll change it.